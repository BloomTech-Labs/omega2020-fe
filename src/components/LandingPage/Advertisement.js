import React from 'react';
import Box from '@material-ui/core/Box';
import Skeleton from '@material-ui/lab/Skeleton';
import CardHeader from '@material-ui/core/CardHeader';
import Button from '@material-ui/core/Button';
import Typography from '@material-ui/core/Typography';
import AppIcon from '../../images/omega-mobile-icon.png';
import BG1 from '../../images/BG1.png';
import { blue } from '@material-ui/core/colors';
import { makeStyles, withStyles } from '@material-ui/core/styles';

const Advertisement = (props) => {
  const classes = useStyles();
  const { loading = false } = props;

  return (
    <Box className={classes.root}>
      <Box className={classes.centerContainer}>
        <Box className={classes.container}>
          <CardHeader
            avatar={
              loading ? (
                <Skeleton className={classes.title} width={600} />
              ) : (
                <img src={AppIcon} alt='App Icon' />
              )
            }
          />
          <CardHeader
            subheader={
              loading ? (
                <Skeleton width={500} height={400} />
              ) : (
                <Typography variant='h4'>Omega 2020</Typography>
              )
            }
          />
          <CardHeader
            subheader={
              loading ? (
                <Skeleton className={classes.title} width={400} height={50} />
              ) : (
                <Typography variant='h5'>
                  Coming soon to app stores near you :&#41;
                </Typography>
              )
            }
          />
          <CardHeader
            subheader={
              loading ? (
                <Skeleton className={classes.text_4} width={60} height={55} />
              ) : (
                <ColorButton
                  variant='contained'
                  color='primary'
                  size='large'
                  href='https://github.com/Lambda-School-Labs/omega2020-fe'
                >
                  <Typography variant='subtitle2'>GitHub</Typography>
                </ColorButton>
              )
            }
          />
        </Box>
      </Box>
    </Box>
  );
};

const useStyles = makeStyles(() => ({
  root: {
    width: '100%',
    height: '65vh',
    display: 'flex',
    flexFlow: 'row wrap',
    justifyContent: 'center',
    textAlign: 'center',
    backgroundImage: `url(${BG1})`,
    backgroundSize: 'cover',
  },
  centerContainer: {
    width: '40%',
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    padding: 30,
  },
  container: {
    width: '100%',
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    alignItems: 'center',
    alignContent: 'center',
    color: 'white',
  },
  title: {
    width: '15%',
    height: 30,
    margin: '5px',
  },
}));

const ColorButton = withStyles((theme) => ({
  root: {
    paddingLeft: 20,
    paddingRight: 20,
    color: theme.palette.getContrastText(blue[600]),
    backgroundColor: blue[600],
    '&:hover': {
      backgroundColor: blue[800],
    },
  },
}))(Button);

export default Advertisement;
