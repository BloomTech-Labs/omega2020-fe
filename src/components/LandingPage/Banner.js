import React from 'react';
import useMediaQuery from '@material-ui/core/useMediaQuery';
import Box from '@material-ui/core/Box';
import Skeleton from '@material-ui/lab/Skeleton';
import CardHeader from '@material-ui/core/CardHeader';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';

import BlueButton from '../assets/BlueButton';
import games from '../../images/games.png';

const Banner = (props) => {
  const classes = useStyles();
  const { loading = false } = props;

  let viewPort = false;
  const md = useMediaQuery('(min-width: 1050px)');
  if (viewPort === md) {
    viewPort = true;
  } else {
    viewPort = false;
  }

  return (
    <Box className={classes.root}>
      <Box className={classes.mainContainer}>
        <CardHeader
          avatar={
            loading ? (
              <Skeleton
                className='bannerIMG'
                variant='rect'
                width={600}
                height={600}
              />
            ) : (
              <Box className={classes.headerBox}>
                {(() => {
                  switch (viewPort) {
                    case true:
                      return (
                        <Box style={{ textAlign: 'center' }}>
                          <Typography
                            className={classes.header}
                            style={{ fontSize: '1.3rem' }}
                            id='improve1'
                          >
                            Improve Your Skills
                            <br />
                            By Understanding Game Strategies
                          </Typography>
                          <br />
                          <Typography
                            className={classes.subheader}
                            id='improve2'
                          >
                            Learn to recognize key patterns and methods
                            <br />
                            to improve your speed and skill level
                          </Typography>
                          <br />
                          <br />
                          <br />
                          <br />
                        </Box>
                      );
                    default:
                      return (
                        <Box style={{ textAlign: 'center' }}>
                          <Typography
                            variant='h4'
                            className={classes.header}
                            id='bang'
                          >
                            Improve Your Skills By Understanding Game Strategies
                          </Typography>
                          <br />
                          <Typography
                            variant='h6'
                            className={classes.subheader}
                          >
                            Learn to recognize key patterns and methods to
                            improve your speed and skill level
                          </Typography>
                          <br />
                          <br />
                          <br />
                          <br />
                        </Box>
                      );
                  }
                })()}
              </Box>
            )
          }
        />
        <CardHeader
          action={
            loading ? (
              <Skeleton className={classes.text} width={60} height={55} />
            ) : (
              <Box>
                <BlueButton title={'Get Started Today'} href={'/tutorial'} />
              </Box>
            )
          }
        />

        <CardHeader
          action={
            loading ? (
              <Skeleton className={classes.text} width={60} height={55} />
            ) : (
              <Box className={classes.gameBoard}>
                {(() => {
                  switch (viewPort) {
                    case true:
                      return (
                        <Box className={classes.img}>
                          {/* <img src={games} alt='games' width='300' /> */}
                        </Box>
                      );
                    default:
                      return (
                        <Box className={classes.img}>
                          {/* <img src={games} alt='games' /> */}
                        </Box>
                      );
                  }
                })()}
              </Box>
            )
          }
        />
      </Box>
    </Box>
  );
};

const useStyles = makeStyles((theme) => ({
  root: {
    width: '100%',
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    alignItems: 'center',
    marginTop: 100,
  },
  mainContainer: {
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    alignItems: 'center',
    textAlign: 'left',
  },
  subheader: {
    // opacity: 0.3,
    color: '#A2A2A2',
  },
  img: {
    width: '90vw',
    height: '50vh',
    backgroundImage: `url(${games})`,
    backgroundRepeat: 'no-repeat',
    // backgroundSize: 'contain, cover',
    backgroundSize: 'cover',
    backgroundPosition: 'center',
  },
  gameBoard: {
    marginTop: 200,
  },
  space: {
    '& > *': {
      margin: theme.spacing(1),
    },
  },
}));

export default Banner;
