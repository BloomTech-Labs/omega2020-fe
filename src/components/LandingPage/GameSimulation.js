import React from 'react';
import useMediaQuery from '@material-ui/core/useMediaQuery';
import Box from '@material-ui/core/Box';
import Skeleton from '@material-ui/lab/Skeleton';
import CardHeader from '@material-ui/core/CardHeader';
import Typography from '@material-ui/core/Typography';
import { blue } from '@material-ui/core/colors';
import { makeStyles } from '@material-ui/core/styles';
import Simulation4x4 from './Simulation4x4';
import Simulation6x6 from './Simulation6x6';

const GameSimulation = (props) => {
  const classes = useStyles();
  const { loading = false } = props;

  let viewPort = false;
  const md = useMediaQuery('(min-width: 1050px)');
  if (viewPort === md) {
    viewPort = true;
  } else {
    viewPort = false;
  }

  return (
    <Box className={classes.root}>
      <Box className={classes.mainContainer}>
        <CardHeader
          avatar={
            loading ? (
              <Skeleton
                className='bannerIMG'
                variant='rect'
                width={600}
                height={600}
              />
            ) : (
              <Box className={classes.headerBox}>
                {(() => {
                  switch (viewPort) {
                    case true:
                      return (
                        <Box style={{ textAlign: 'center' }}>
                          <Typography
                            className={classes.header}
                            style={{ fontSize: '1.3rem' }}
                            id='learn'
                          >
                            Learn The Game Play
                          </Typography>
                          <br />
                          <Typography className={classes.subheader} id='start'>
                            Start off with something easy
                            <br />
                            then kick it up a notch
                          </Typography>
                          <br />
                          <br />
                          <br />
                          <br />
                        </Box>
                      );
                    default:
                      return (
                        <Box style={{ textAlign: 'center' }}>
                          <Typography
                            variant='h4'
                            className={classes.header}
                            id='bang'
                          >
                            Learn The Game Play
                          </Typography>
                          <br />
                          <Typography
                            variant='h6'
                            className={classes.subheader}
                          >
                            Start off with something easy then kick it up a
                            notch
                          </Typography>
                          <br />
                          <br />
                          <br />
                          <br />
                        </Box>
                      );
                  }
                })()}
              </Box>
            )
          }
        />
        <CardHeader
          avatar={
            loading ? (
              <Skeleton
                className='bannerIMG'
                variant='rect'
                width={600}
                height={600}
              />
            ) : (
              <Box className={classes.Simulation}>
                <Simulation4x4 />
                <br />
                <Simulation6x6 />
              </Box>
            )
          }
        />
      </Box>
    </Box>
  );
};

const useStyles = makeStyles(() => ({
  root: {
    width: '100%',
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    alignItems: 'center',
    marginTop: 200,
  },
  mainContainer: {
    width: '100%',
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    alignItems: 'center',
    textAlign: 'left',
  },
  subheader: {
    // opacity: 0.3,
    color: '#A2A2A2',
  },
  Simulation: {
    width: '100%',
  },
}));

export default GameSimulation;
