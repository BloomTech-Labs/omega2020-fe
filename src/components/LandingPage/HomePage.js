import React from 'react';
import useMediaQuery from '@material-ui/core/useMediaQuery';
import Box from '@material-ui/core/Box';
import Skeleton from '@material-ui/lab/Skeleton';
import CardHeader from '@material-ui/core/CardHeader';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import Tooltip from '@material-ui/core/Tooltip';
import { grey } from '@material-ui/core/colors';
import { makeStyles } from '@material-ui/core/styles';

import CategoryIcon from '@material-ui/icons/Category';
import AppsIcon from '@material-ui/icons/Apps';
import GetAppIcon from '@material-ui/icons/GetApp';

import BlueButton from '../assets/BlueButton';
import Cube from '../../images/cube4.gif';

const HomePage = (props) => {
  const classes = useStyles();
  const { loading = false } = props;

  let viewPort = false;
  const md = useMediaQuery('(min-width: 1050px)');
  if (viewPort === md) {
    viewPort = true;
  } else {
    viewPort = false;
  }

  return (
    <Box className={classes.root}>
      <Box className={classes.mainContainer}>
        <CardHeader
          avatar={
            loading ? (
              <Skeleton
                className='bannerIMG'
                variant='rect'
                width={600}
                height={600}
              />
            ) : (
              <Box className={classes.headerBox}>
                {(() => {
                  switch (viewPort) {
                    case true:
                      return (
                        <Box>
                          <Typography
                            className={classes.header}
                            style={{ fontSize: '1.3rem' }}
                            id='sudo'
                          >
                            Sudomega Interactive Sudoku
                          </Typography>
                          <br />
                          <Typography className={classes.subheader} id='upload'>
                            Learn how to play, upload your own sudoku
                            <br />
                            games, and compete with friends :&#41;
                          </Typography>
                        </Box>
                      );
                    default:
                      return (
                        <Box>
                          <Typography variant='h3' className={classes.header}>
                            Sudomega Interactive Sudoku
                          </Typography>
                          <br />
                          <Typography
                            variant='body1'
                            className={classes.subheader}
                          >
                            Learn how to play, upload your own sudoku
                            <br />
                            games, and compete with friends :&#41;
                          </Typography>
                        </Box>
                      );
                  }
                })()}
              </Box>
            )
          }
        />
        <CardHeader
          action={
            loading ? (
              <Skeleton className={classes.text} width={60} height={55} />
            ) : (
              <Box>
                <br />
                <br />
                <Box className={classes.space}>
                  <BootstrapTooltip title='Get Started'>
                    <IconButton color='inherit' href={'/tutorial'}>
                      <CategoryIcon className={classes.icon} />
                    </IconButton>
                  </BootstrapTooltip>
                  <BootstrapTooltip title='Play'>
                    <IconButton color='inherit' href={'/levels'}>
                      <AppsIcon className={classes.icon} />
                    </IconButton>
                  </BootstrapTooltip>
                  <BootstrapTooltip title='Upload'>
                    <IconButton color='inherit' href={'/upload'}>
                      <GetAppIcon className={classes.icon} />
                    </IconButton>
                  </BootstrapTooltip>
                </Box>
                <br />
                <br />
              </Box>
            )
          }
        />
        <CardHeader
          action={
            loading ? (
              <Skeleton className={classes.text} width={60} height={55} />
            ) : (
              <Box>
                <BlueButton title={'Get Started'} href={'/tutorial'} />
              </Box>
            )
          }
        />

        <CardHeader
          action={
            loading ? (
              <Skeleton className={classes.text} width={60} height={55} />
            ) : (
              <Box className={classes.gameBoard}>
                {/* <RenderPuzzle /> */}
                <img src={Cube} alt={'cibe'} width={300} />
              </Box>
            )
          }
        />
      </Box>
    </Box>
  );
};

const useStylesBootstrap = makeStyles((theme) => ({
  arrow: {
    color: grey[200],
  },
  tooltip: {
    backgroundColor: grey[200],
    color: '#000',
  },
}));

function BootstrapTooltip(props) {
  const classes = useStylesBootstrap();

  return <Tooltip arrow classes={classes} {...props} />;
}

const useStyles = makeStyles((theme) => ({
  root: {
    width: '100%',
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    alignItems: 'center',
    marginTop: 100,
    marginBotom: 100,
  },
  mainContainer: {
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    alignItems: 'center',
    textAlign: 'center',
  },
  subheader: {
    // opacity: 0.3,
    // color: '#A2A2A2',
  },
  icon: {
    fontSize: 80,
    color: grey[700],
  },
  gameBoard: {
    marginTop: 100,
  },
  space: {
    '& > *': {
      margin: theme.spacing(1),
    },
  },
}));

export default HomePage;
