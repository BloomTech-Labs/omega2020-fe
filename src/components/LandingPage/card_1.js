import React from 'react';
import Box from '@material-ui/core/Box';
import Skeleton from '@material-ui/lab/Skeleton';
import CardHeader from '@material-ui/core/CardHeader';
import Typography from '@material-ui/core/Typography';
import Button from '@material-ui/core/Button';
import grid4x4 from '../../images/4x4-grid.png';
import { blue } from '@material-ui/core/colors';
import { makeStyles, withStyles } from '@material-ui/core/styles';

const Home = (props) => {
  const classes = useStyles();
  const { loading = false } = props;

  return (
    <Box className={classes.root}>
      <Box className={classes.Container}>
        <Box className={classes.container}>
          <CardHeader
            title={
              loading ? (
                <Skeleton
                  className={classes.title}
                  width={600}
                  height={100}
                  m={0}
                  p={0}
                />
              ) : (
                <Typography variant='h4'>
                  Letâ€™s start off with something easy
                </Typography>
              )
            }
          />
          <CardHeader
            title={
              loading ? (
                <Skeleton
                  className={classes.title}
                  width={600}
                  height={100}
                  m={0}
                  p={0}
                />
              ) : (
                <Box>
                  <Typography variant='h4'>
                    Try to solve this 4x4 sudoku:
                  </Typography>
                  <Typography variant='subtitle1'>
                    Each row, column and square needs to be filled out with the
                    numbers 1-4, without repeating any numbers within the row,
                    column or square.
                  </Typography>
                </Box>
              )
            }
          />
          <CardHeader
            subheader={
              loading ? (
                <Skeleton width={500} height={400} margin={0} padding={0} />
              ) : (
                <img src={grid4x4} alt='game' />
              )
            }
          />
          <CardHeader
            subheader={
              loading ? (
                <Skeleton className={classes.text_4} width={60} height={55} />
              ) : (
                <Box>
                  <ColorButton variant='contained' color='primary'>
                    Reveal Answer
                  </ColorButton>
                </Box>
              )
            }
          />
        </Box>
      </Box>
    </Box>
  );
};

const useStyles = makeStyles((theme) => ({
  root: {
    display: 'flex',
    flexFlow: 'row wrap',
    justifyContent: 'space-evenly',
    background: '#000000',
    color: '#FFFFFF',
    padding: 90,
  },
  Container: {
    width: '40%',
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    padding: 30,
  },
  container: {
    width: '100%',
  },
  title: {
    width: '40%',
    display: 'flex',
    flexFlow: 'column wrap',
    justifyContent: 'center',
    border: '2px solid red',
  },
}));

const ColorButton = withStyles((theme) => ({
  root: {
    paddingLeft: 20,
    paddingRight: 20,
    color: theme.palette.getContrastText(blue[600]),
    backgroundColor: blue[600],
    '&:hover': {
      backgroundColor: blue[800],
    },
  },
}))(Button);

export default Home;
