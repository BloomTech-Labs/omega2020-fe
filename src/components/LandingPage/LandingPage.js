import React from 'react';
import Box from '@material-ui/core/Box';
import Container from '@material-ui/core/Container';
import HomePage from './HomePage';
import GameSimulation from './GameSimulation';
import Banner from './Banner';

const LandingPage = (props) => {
  return (
    <Box>
      <Container>
        <HomePage theme={props.theme} />
        <GameSimulation />
      </Container>
      <Banner />
    </Box>
  );
};

export default LandingPage;
