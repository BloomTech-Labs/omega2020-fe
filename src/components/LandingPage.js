import React from 'react';
import Banner from './LandingPage/Banner';
import Info from './LandingPage/Info';
import card_1 from './LandingPage/card_1';

const LandingPage = () => {
  return (
    <React.Fragment>
      <Banner />
      <Info />
      <card_1 />
    </React.Fragment>
  );
};

export default LandingPage;
