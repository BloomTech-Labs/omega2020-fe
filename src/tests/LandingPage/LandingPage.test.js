import React from 'react';
import renderer from 'react-test-renderer';
import { configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import '@testing-library/jest-dom/extend-expect';
import HomePage from '../../components/LandingPage/HomePage';
import Banner from '../../components/LandingPage/Banner';
import GameSimulation from '../../components/LandingPage/GameSimulation';
import LandingPage from '../../components/LandingPage/LandingPage';

configure({ adapter: new Adapter() });

describe('the landing page info', () => {
    const page = mount(<LandingPage />);
    
    const renderPage = renderer.create(<LandingPage />);
    const rootPage = renderPage.root;

    it('snapshot the components', () => {
        const shot = renderer.create(<LandingPage />).toJSON();
        expect(shot).toMatchSnapshot();
    })

    it('checks the props in the HomePage component, the length of the GameSimulation, and Banner component ', () => {
        expect(page.find(HomePage).length).toEqual(1);
        expect(page.find(Banner).length).toEqual(1);
        expect(page.find(GameSimulation).length).toEqual(1);

        expect(rootPage.findByType(HomePage).props.theme).toBe(undefined)
    })
})