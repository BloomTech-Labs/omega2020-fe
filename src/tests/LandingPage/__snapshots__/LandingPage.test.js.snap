// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the landing page info snapshot the components 1`] = `
<div
  className="MuiBox-root MuiBox-root-93"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthLg"
  >
    <div
      className="MuiBox-root MuiBox-root-94 makeStyles-root-2"
    >
      <div
        className="MuiBox-root MuiBox-root-95 makeStyles-mainContainer-3"
      >
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-avatar"
          >
            <div
              className="MuiBox-root MuiBox-root-96"
            >
              <div
                className="MuiBox-root MuiBox-root-97"
              >
                <p
                  className="MuiTypography-root MuiTypography-body1"
                  id="sudo"
                  style={
                    Object {
                      "fontSize": "1.3rem",
                    }
                  }
                >
                  Sudomega Interactive Sudoku
                </p>
                <br />
                <p
                  className="MuiTypography-root makeStyles-subheader-4 MuiTypography-body1"
                  id="upload"
                >
                  Learn how to play, upload your own sudoku
                  <br />
                  games, and compete with friends :)
                </p>
              </div>
            </div>
          </div>
          <div
            className="MuiCardHeader-content"
          />
        </div>
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content"
          />
          <div
            className="MuiCardHeader-action"
          >
            <div
              className="MuiBox-root MuiBox-root-98"
            >
              <br />
              <br />
              <div
                className="MuiBox-root MuiBox-root-99 makeStyles-space-7"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root makeStyles-icon-5"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 2l-5.5 9h11z"
                  />
                  <circle
                    cx="17.5"
                    cy="17.5"
                    r="4.5"
                  />
                  <path
                    d="M3 13.5h8v8H3z"
                  />
                </svg>
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root makeStyles-icon-5"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M4 8h4V4H4v4zm6 12h4v-4h-4v4zm-6 0h4v-4H4v4zm0-6h4v-4H4v4zm6 0h4v-4h-4v4zm6-10v4h4V4h-4zm-6 4h4V4h-4v4zm6 6h4v-4h-4v4zm0 6h4v-4h-4v4z"
                  />
                </svg>
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root makeStyles-icon-5"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 9h-4V3H9v6H5l7 7 7-7zM5 18v2h14v-2H5z"
                  />
                </svg>
              </div>
              <br />
              <br />
            </div>
          </div>
        </div>
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content"
          />
          <div
            className="MuiCardHeader-action"
          >
            <div
              className="MuiBox-root MuiBox-root-100"
            >
              <a
                aria-disabled={false}
                className="MuiButtonBase-root MuiButton-root WithStyles(ForwardRef(Button))-root-15 MuiButton-text href MuiButton-textSizeLarge MuiButton-sizeLarge"
                href="/tutorial"
                onBlur={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                tabIndex={0}
              >
                <span
                  className="MuiButton-label"
                >
                  <p
                    className="MuiTypography-root title MuiTypography-body2"
                  >
                    Get Started; learn the game play
                  </p>
                </span>
                <span
                  className="MuiTouchRipple-root"
                />
              </a>
              <br />
              <br />
              <span
                className="MuiTypography-root MuiTypography-caption"
              >
                No signup required
              </span>
            </div>
          </div>
        </div>
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-content"
          />
          <div
            className="MuiCardHeader-action"
          >
            <div
              className="MuiBox-root MuiBox-root-101 makeStyles-gameBoard-6"
            >
              <div
                className="makeStyles-root-17"
              >
                <div>
                  <div
                    className="makeStyles-board-18"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="MuiBox-root MuiBox-root-102 makeStyles-root-20"
    >
      <div
        className="MuiBox-root MuiBox-root-103 makeStyles-mainContainer-21"
      >
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-avatar"
          >
            <div
              className="MuiBox-root MuiBox-root-104"
            >
              <div
                className="MuiBox-root MuiBox-root-105"
                style={
                  Object {
                    "textAlign": "center",
                  }
                }
              >
                <p
                  className="MuiTypography-root MuiTypography-body1"
                  id="learn"
                  style={
                    Object {
                      "fontSize": "1.3rem",
                    }
                  }
                >
                  Learn The Game Play
                </p>
                <br />
                <p
                  className="MuiTypography-root makeStyles-subheader-22 MuiTypography-body1"
                  id="start"
                >
                  Start off with something easy
                  <br />
                  then kick it up a notch
                </p>
                <br />
                <br />
                <br />
                <br />
              </div>
            </div>
          </div>
          <div
            className="MuiCardHeader-content"
          />
        </div>
        <div
          className="MuiCardHeader-root"
        >
          <div
            className="MuiCardHeader-avatar"
          >
            <div
              className="MuiBox-root MuiBox-root-106 makeStyles-Simulation-23"
            >
              <div
                className="MuiBox-root MuiBox-root-107 makeStyles-root-29"
              >
                <div
                  className="MuiBox-root MuiBox-root-108 makeStyles-mainContainer-30"
                >
                  <div
                    className="MuiBox-root MuiBox-root-109"
                    style={
                      Object {
                        "textAlign": "center",
                      }
                    }
                  >
                    <p
                      className="MuiTypography-root MuiTypography-body1"
                      id="solve"
                      style={
                        Object {
                          "fontSize": "1.3rem",
                        }
                      }
                    >
                      Try to solve this 4x4 sudoku:
                    </p>
                    <br />
                    <p
                      className="MuiTypography-root makeStyles-subheader-31 MuiTypography-body1"
                      id="row"
                    >
                      Each row, column and square needs to be filled
                      <br />
                      out with the numbers 1-4, without repeating any
                      <br />
                      numbers within the row, column or square.
                    </p>
                    <br />
                    <div
                      className="MuiBox-root MuiBox-root-110"
                    >
                      <img
                        alt="Game4x4"
                        src="Game4x4.png"
                        width="300"
                      />
                    </div>
                  </div>
                </div>
              </div>
              <br />
              <div
                className="MuiBox-root MuiBox-root-111 makeStyles-root-40"
              >
                <div
                  className="MuiBox-root MuiBox-root-112 makeStyles-mainContainer-41"
                >
                  <div
                    className="MuiBox-root MuiBox-root-113"
                    style={
                      Object {
                        "textAlign": "center",
                      }
                    }
                  >
                    <p
                      className="MuiTypography-root MuiTypography-body1"
                      id="solve"
                      style={
                        Object {
                          "fontSize": "1.3rem",
                        }
                      }
                    >
                      Try to solve this 6x6 sudoku:
                    </p>
                    <br />
                    <p
                      className="MuiTypography-root makeStyles-subheader-42 MuiTypography-body1"
                      id="row"
                    >
                      Each row, column and square needs to be filled
                      <br />
                      out with the numbers 1-6, without repeating any
                      <br />
                      numbers within the row, column or square.
                    </p>
                    <br />
                    <div
                      className="MuiBox-root MuiBox-root-114"
                    >
                      <img
                        alt="Game6x6"
                        src="Game6x6.png"
                        width="300"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            className="MuiCardHeader-content"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiBox-root MuiBox-root-115 makeStyles-root-51"
  >
    <div
      className="MuiBox-root MuiBox-root-116 makeStyles-mainContainer-52"
    >
      <div
        className="MuiCardHeader-root"
      >
        <div
          className="MuiCardHeader-avatar"
        >
          <div
            className="MuiBox-root MuiBox-root-117"
          >
            <div
              className="MuiBox-root MuiBox-root-118"
              style={
                Object {
                  "textAlign": "center",
                }
              }
            >
              <p
                className="MuiTypography-root MuiTypography-body1"
                id="improve1"
                style={
                  Object {
                    "fontSize": "1.3rem",
                  }
                }
              >
                Improve Your Skills
                <br />
                By Understanding Game Strategies
              </p>
              <br />
              <p
                className="MuiTypography-root makeStyles-subheader-53 MuiTypography-body1"
                id="improve2"
              >
                Learn to recognize key patterns and methods
                <br />
                to improve your speed and skill level
              </p>
              <br />
              <br />
              <br />
              <br />
            </div>
          </div>
        </div>
        <div
          className="MuiCardHeader-content"
        />
      </div>
      <div
        className="MuiCardHeader-root"
      >
        <div
          className="MuiCardHeader-content"
        />
        <div
          className="MuiCardHeader-action"
        >
          <div
            className="MuiBox-root MuiBox-root-119"
          >
            <a
              aria-disabled={false}
              className="MuiButtonBase-root MuiButton-root WithStyles(ForwardRef(Button))-root-15 MuiButton-text href MuiButton-textSizeLarge MuiButton-sizeLarge"
              href="/tutorial"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
            >
              <span
                className="MuiButton-label"
              >
                <p
                  className="MuiTypography-root title MuiTypography-body2"
                >
                  Get Started Today
                </p>
              </span>
              <span
                className="MuiTouchRipple-root"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        className="MuiCardHeader-root"
      >
        <div
          className="MuiCardHeader-content"
        />
        <div
          className="MuiCardHeader-action"
        >
          <div
            className="MuiBox-root MuiBox-root-120 makeStyles-gameBoard-55"
          >
            <div
              className="MuiBox-root MuiBox-root-121 makeStyles-img-54"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
